udata/models/owned.py

```diff
@@ -43,6 +43,9 @@ def owned_pre_save(sender, document, **kwargs):
     if not isinstance(document, Owned):
         return
     changed_fields = getattr(document, '_changed_fields', [])
+    if 'organization' in changed_fields and 'owner' in changed_fields:
+        # Ownership changes (org to owner or the other way around) have already been made
+        return
     if 'organization' in changed_fields:
```

Comment:
While we're at it, if we go towards less business logic in signal handling, this whole `_previous_owner` seems overkill at first glance if it's only used here https://github.com/opendatateam/udata/blob/86173677392abedc532aabf1b78e842496c81f2e/udata/models/owned.py#L68. Why not send `on_owner_change` directly from here?

Reply:
Forget it, it's probably because we want `post_save` :-) But still what a mess 🍝